src/WebJobs.Script/Diagnostics/Extensions/ExtensionBundleLoggerExtension.cs

```diff
@@ -83,6 +84,26 @@ internal static class ExtensionBundleLoggerExtension
             new EventId(111, nameof(MatchingBundleNotFound)),
             "Bundle version matching the {version} was not found");
 
+        private static readonly Action<ILogger, Uri, HttpStatusCode?, HttpRequestError?, string, string, Exception> _errorDownloadingZipWithDiskSize =
+            LoggerMessage.Define<Uri, HttpStatusCode?, HttpRequestError?, string, string>(
+            LogLevel.Error,
+            new EventId(112, nameof(ErrorDownloadingZipDiskSize)),
+            "Error downloading Zip content {zip}. Status Code:{statusCode} RequestError:{RequestError} FilePath:{FilePath} Disk:{DiskUsage}");
```

Comment:
This will show up in the custom dimensions. Let's apply camel casing consistently across all attributes.

Reply:
Thanks. all the method similar one, I changed it into camel.